BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/Skonto/Views/SkontoProceedView.swift

```diff
@@ -0,0 +1,174 @@
+//
+//  SkontoProceedView.swift
+//
+//  Copyright © 2024 Gini GmbH. All rights reserved.
+//
+
+import UIKit
+import GiniCaptureSDK
+
+class SkontoProceedView: UIView {
+    private lazy var payButton: MultilineTitleButton = {
+        let button = MultilineTitleButton()
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.configure(with: configuration.primaryButtonConfiguration)
+        button.titleLabel?.font = configuration.textStyleFonts[.bodyBold]
+        let buttonTitle = NSLocalizedStringPreferredGiniBankFormat("ginibank.digitalinvoice.paybutton.title",
```

Comment:
[suggestion] We should use a dedicated localizabale string for each new text for skonto.